src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -81,7 +81,7 @@ public class SalaryInfoTest {
 
     @Test
     public void getSalaryInfoByMonth() {
-        SalaryInfo salary = new SalaryInfo();
+        SalaryInfoProvider salary = new SalaryInfoProvider();
```

Comment:
okay, it seems my advice on changing the class name is not valid :)

you should not change the previously written tests - you can rollback to previous classname


sorry to inconvenience :)